test_workloads.sh

```diff
@@ -12,9 +13,21 @@ run_test() {
   local workload_name=$2
   local db=$3
   local test_name_suffix=${workload_name}_${db}_${n_clients}
+  local controller=$4
+  
+  # setup controller path
+  if [ $controller == gdpr ]; then
+    controller_path=GDPRuler.py
+  elif [ $controller == native ]; then
+    controller_path=native_ctl.py
+  fi
+
+  # clear logs
+  rm -rf logs/
 
   if [ $db == rocksdb ]; then
     # run rocksdb server
+    rm -rf ./db
```

Comment:
Adding a configuration parameter for the DB file would be better so that we can place it directly in the /scratch/ location of the local disk.

Reply:
done